### ISSUE TITLE
Play/Pause Button not working.

### ISSUE THREAD

**Issue by raphcrf** [xzhih/dell-7460-7560-hackintosh#52]
Hello there.

I was wondering if there was a workaround for the Play/Pause not working.

When i first installed, it was working 100%, but after a few days, it just stopped.

It's the only media key that doesn't work.

Thanks in advance!

**Comment by raphcrf** [comment]
Yes i did, it's Karabiner's event viewer i'm using to check if the key works.

On windows it works normally. ;/

**Comment by xzhih** [comment]
Try using the latest VoodooPS2Controller.kext, I don't know if it doesn't work.

Download [here](https://bitbucket.org/RehabMan/os-x-voodoo-ps2-controller/downloads/).

**Comment by raphcrf** [comment]
I've compared both versions, the one i got from here and this one you mentioned. The Rehab Version seems outdated somehow, i don't know. But i'll try anyways.

I'll come back with feedback.

**Comment by raphcrf** [comment]
Tried it, but still not working.

It's like the key is not recognized by the system at all.

**Comment by xzhih** [comment]
Only the Play/Pause key?
🤔🤔🤔

**Comment by raphcrf** [comment]
Yes, unfortunately. If you look to the keyboard, it's the F5 key.

If i tell Mac to use the F keys automatically as function keys or press it with Fn, it works like a normal f5 key, but using it as a media button without fn key, it simply doesn't work. ;/

It worked when i first installed, i don't know what went wrong.

**Comment by xzhih** [comment]
I don't know either. 😂

**Comment by the-eric-kwok** [comment]
Have this issue on my HP laptop too! I can see `sending key e022=34 down` in my console but not in Karabiner-EventViewer.
